src/route/nodes/netid/nodeEdit/NodeEdit.tsx

```diff
@@ -283,8 +283,8 @@ export const NodeEdit: React.FC<{
           <Grid container justifyContent="space-between" alignItems="center">
             <Grid item xs={10} sm={4} md={2} sx={rowMargin}>
               <NmFormInputSwitch
-                label={String(t('node.isstatic'))}
-                name={'isstatic'}
+                label={String(t('node.dynamicendpoint'))}
+                name={'dynamicendpoint'}
```

Comment:
I believe this should remain isstatic in order to parse the value correctly.

Reply:
ok fixed that